cmd/kube-scheduler/app/options/options.go

```diff
@@ -211,6 +214,16 @@ func (o *Options) ApplyTo(c *schedulerappconfig.Config) error {
 	return nil
 }
 
+// emptySchedulerProfileConfig returns true if the list of profiles passed to it contains only
+// the "default-scheduler" profile with no plugins or pluginconfigs registered
+// (this is the default empty profile initialized by defaults.go)
+func emptySchedulerProfileConfig(profiles []kubeschedulerconfig.KubeSchedulerProfile) bool {
+	return len(profiles) == 1 &&
+		len(profiles[0].PluginConfig) == 0 &&
+		profiles[0].Plugins == nil &&
+		profiles[0].SchedulerName == "default-scheduler"
```

Comment:
You should still be able to set a schedulerName

Reply:
removed the check for `profiles[0].SchedulerName == "default-scheduler"`